mathesar/models/base.py

```diff
@@ -921,6 +922,11 @@ def _set_default_preview_template(table):
 
 
 def compute_default_preview_template(table):
+    if table.id in CACHED_PREVIEW_FOR_TABLES:
+        cached_preview_column = CACHED_PREVIEW_FOR_TABLES[table.id]
+        if Column.current_objects.filter(id=cached_preview_column):
+            preview_template = f"{{{cached_preview_column}}}"
+            return preview_template
```

Comment:
You've introduced a cache and you're performing two operations on it: setting and getting. Make those into discrete methods to improve maintainability.

Reply:
Sure, I will do that.